resources/views/auth/user.blade.php

```diff
@@ -6,14 +6,155 @@
         <div class="card">
             <div class="card-content">
                 <div class="card-title">@lang('info.user_data')</div>
-                <p>Your user data will appear here</p>
-                <!--TODO-->
+                <table>
```

Comment:
Does the new UI rely on tables instead of divs? I prefer using divs as they can be configured to look good on all devices.

Reply:
I tested this on mobiles, it's easier for now, but I agree. (btw there is a responsive-table option in materialize, that swaps columns and rows on smaller sizes, but that's horrible in this case)